src/pages/About/About.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  IonCard,
+  IonCardContent,
+  IonCardHeader,
+  IonGrid,
+  IonCardTitle,
+} from '@ionic/react';
+
+const About: React.FC = () => {
+  return (
+    <IonPage>
+      <IonHeader className="ion-hide-sm-up" mode="md">
+        <IonToolbar>
+          <IonTitle>About</IonTitle>
+        </IonToolbar>
+      </IonHeader>
+      <IonContent>
+        <IonGrid className="page-header" fixed>
+          <IonCard>
+            <img
+              alt="barcelona"
+              style={{ height: '150px', objectFit: 'cover', width: '100%' }}
```

Comment:
maybe create a cssc class?

Reply:
good point, but since it was only there for demo purposes I will just clean the whole page, and leave it empty.